ml_datasets/loaders/mnist.py

```diff
@@ -28,10 +31,34 @@ def mnist(variant='mnist', shuffle=True):
         (X_train, y_train), (X_test, y_test) = load_fashion_mnist()
     elif variant == 'kuzushiji':
         (X_train, y_train), (X_test, y_test) = load_kuzushiji_mnist()
+    elif variant.startswith("emnist-"):
+        split = variant.split("-")[1]
+        if split not in [
+                'digits', 'letters',
+                'balanced', 'byclass',
+                'bymerge', 'mnist'
+        ]:
+            raise ValueError(
+                "To load EMNIST use the format "
+                "'emnist-split' where 'split' can be"
+                "'digits', 'letters', 'balanced' "
+                "'byclass', 'bymerge' and 'mnist'."
+            )
+        else:
+            (X_train, y_train), (X_test, y_test) = load_emnist(split=split)
     else:
-        raise ValueError("Variant must be one of: 'mnist', 'fashion', 'kuzushiji'")
-    X_train = X_train.reshape(60000, 784)
-    X_test = X_test.reshape(10000, 784)
+        raise ValueError(
+            "Variant must be one of: "
+            "'mnist', 'fashion', 'kuzushiji' "
+            "'emnist-digits', 'emnist-letters' "
+            "'emnist-balanced', 'emnist-byclass' "
+            "'emnist-bymerge', 'emnist-mnist'."
```

Comment:
```suggestion
            "Variant must be one of: "
            "'mnist', 'fashion', 'kuzushiji', "
            "'emnist-digits', 'emnist-letters', "
            "'emnist-balanced', 'emnist-byclass', "
            "'emnist-bymerge', 'emnist-mnist'."
```
+ `black` formatting